CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ### New features:
 * Add modules for first and second order Fermi acceleration
+* Finalized availability of custom photon fields in the PhotoPionProduction.
```

Comment:
For consistency with the way in which the changelog is being done (to make it possible to copy-paste it when we release the next version without any edits), I'd suggest to be less verbose here: "Make custom photon fields available for photopion production" or "Make custom photon fields available in the PhotoPionProduction module."